models/demos/tg/llama3_70b/tests/test_llama_attention_galaxy.py

```diff
@@ -204,7 +204,7 @@ def tt_llama_attention_prepare_inputs(llama_attention_model, x, start_pos, rope_
         )
 
         cos, sin = precompute_freqs(
-            llama_attention_model.head_dim, llama_attention_model.max_seq_len * 2, rope_theta, use_scaled=True
+            llama_attention_model.head_dim, llama_attention_model.max_seq_len * 2, rope_theta, use_scaled=False
```

Comment:
This is still with llama3 weights. We pushed llama3.1 weights but since it's needed to update reference code, decode and prefill implementation I decided to add it in separate PR.